test/snippet/alphabet/structure/dot_bracket3_char_literal.cpp

```diff
@@ -0,0 +1,12 @@
+#include <seqan3/alphabet/structure/dot_bracket3.hpp>
+
+int main()
+{
+    using seqan3::operator""_db3;
+    
+    // Using the char literal to assign a single dot bracket:
+    seqan3::dot_bracket3 my_letter{'('_db3};
+    // does not work:
+    // seqan3::dot_bracket3 my_letter{'('}; // <- char not implicitly convertible
```

Comment:
just make a search for "work" to find all these snippets maybe.

Reply:
There is a snippet, `all_ambiguous.cpp` which is literally just a "does not work". Should I just delete the snippet?

Also, there are some which are more of "This throws an exception". Should those be kept?

Reply:
(I removed all, including the ones that said something along the lines of "Doing this throws an exception", except the one mentioned above because it was the entire snippet.)